.github/workflows/ci.yml

```diff
@@ -60,3 +60,22 @@ jobs:
         run: |
           curl -o codecov.sh -f https://codecov.io/bash || curl -o codecov.sh -f https://codecov.io/bash || curl -o codecov.sh -f https://codecov.io/bash
           bash codecov.sh -n "tox -e ${{ matrix.PYTHON.TOXENV }}"
+
+  linux-docker:
+    runs-on: ubuntu-latest
+    container: ghcr.io/pyca/cryptography-runner-${{ matrix.TEST.CONTAINER }}
+    strategy:
+      matrix:
+        TEST:
+          - {CONTAINER: "stretch", TOXENV: "py27"}
```

Comment:
lol two Pythons cryptography is dropping. I guess this is good to ensure we don't drop it unintentionally from pyOpenSSL until we're ready.